scripts/create_tables.sql

```diff
@@ -1 +1,30 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  specialty TEXT
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  insurance TEXT
+);
+
+CREATE TABLE appointments (
+  id SERIAL PRIMARY KEY,
+  appointment_date DATE,
+  appointment_time TIME,
+  comments TEXT,
+  patient_id INTEGER REFERENCES patients(id),
+  doctor_id INTEGER REFERENCES doctors(id)
+);
+
+CREATE INDEX ON appointments(patient_id);
+CREATE INDEX ON appointments(doctor_id);
```

Comment:
Good, we don't need to CREATE INDEX here at the bottom either, we already set up the columns with the REFERENCES lines.